ydb/public/sdk/cpp/examples/vector_index/README.md

```diff
@@ -0,0 +1,71 @@
+# Vector Index
+
+
+Parameters description
+
+```
+./vector_index --help
+Usage: ./vector_index [OPTIONS] [ARG]...
+
+Required parameters:
+  {-e|--endpoint} HOST:PORT YDB endpoint
+  {-d|--database} PATH      YDB database
+  {-c|--command} COMMAND    execute command: [Create|Drop|Build|Recreate]Index
+                            or TopK (used for search, read request)
+  --table TABLE             table name
+  --index_type TYPE         index type: [flat]
+  --index_quantizer QUANTIZER
+                            index quantizer: [none|int8|uint8|bit]
+  --primary_key PK          primary key column
+  --embedding EMBEDDING     embedding (vector) column
+  --distance DISTANCE       distance function:
+                            [Cosine|Euclidean|Manhattan]Distance
+  --rows ROWS               count of rows in table, used only for
+                            [Build|Recreate]Index commands
+  --top_k TOPK              count of rows in top, used only for TopK command
+  --data DATA               list of columns to read, used only for TopK command
+  --target TARGET           file with target vector, used only for TopK command
+```
+
+RecreateIndex -- command which sequentially executes Drop, Create and Build commands
+
+Index table will have name like `<table>_<index_type>_<index_quantizer>`
+
+## Examples
+
+### Flat Index
+
+It uses scalar quantization to speedup ANN search
```

Comment:
Please add more text here.

Reply:
What text do you suggest? I described how it works in a few lines below

Reply:
Approximate search algorithm:

* an approximate search is performed using bit quantization;
* an approximate list of vectors is obtained;
* we search this list without using quantization.